code/modules/events/vent_clog.dm

```diff
@@ -1,49 +1,39 @@
 /datum/round_event_control/vent_clog
 	name = "Clogged Vents"
 	typepath = /datum/round_event/vent_clog
-	weight = 35
+	weight = 55
 
 /datum/round_event/vent_clog
 	announceWhen	= 1
 	startWhen		= 5
 	endWhen			= 35
 	var/interval 	= 2
 	var/list/vents  = list()
-	var/list/gunk = list("water","carbon","flour","radium","toxin","cleaner","nutriment","condensedcapsaicin","mushroomhallucinogen","lube",
-								 "plantbgone","banana","charcoal","space_drugs","morphine","holywater","ethanol","hot_coco","sacid")
-
-/datum/round_event/vent_clog/announce(fake)
+/datum/round_event/vent_clog/announce()
 	priority_announce("The scrubbers network is experiencing a backpressure surge. Some ejection of contents may occur.", "Atmospherics alert")
 
 
 /datum/round_event/vent_clog/setup()
 	endWhen = rand(25, 100)
 	for(var/obj/machinery/atmospherics/components/unary/vent_scrubber/temp_vent in GLOB.machines)
 		if(is_station_level(temp_vent.loc.z) && !temp_vent.welded)
-			var/datum/pipeline/temp_vent_parent = temp_vent.parents[1]
-			if(temp_vent_parent.other_atmosmch.len > 20)
-				vents += temp_vent
+			vents += temp_vent
 	if(!vents.len)
 		return kill()
 
-/datum/round_event/vent_clog/tick()
-	if(activeFor % interval == 0)
-		var/obj/machinery/atmospherics/components/unary/vent = pick_n_take(vents)
-		while(vent && vent.welded)
-			vent = pick_n_take(vents)
-
+/datum/round_event/vent_clog/start()
+	for(var/obj/machinery/atmospherics/components/unary/vent in vents)
 		if(vent && vent.loc)
-			var/datum/reagents/R = new/datum/reagents(50)
+			var/datum/reagents/R = new/datum/reagents(1000)
 			R.my_atom = vent
-			R.add_reagent(pick(gunk), 50)
+			R.add_reagent(get_random_reagent_id(), 1000)

```

Comment:
This doesn't look like 4000u to me

Reply:
then fix it and make your own pr

Reply:
Smoke/foam carry 4x the reagents now if I'm correct

Reply:
Oh!

Reply:
@JJRcop  #34485

Reply:
All is well!

When will github allow you to cancel a review though :thinking: